wfe/wfe.go

```diff
@@ -1775,6 +1846,91 @@ func (wfe *WebFrontEndImpl) orderForDisplay(
 	return result
 }
 
+// RenewalInfo implements ACME Renewal Info (ARI)
+func (wfe *WebFrontEndImpl) RenewalInfo(_ context.Context, response http.ResponseWriter, request *http.Request) {
```

Comment:
Why take a context here at all?

```suggestion
func (wfe *WebFrontEndImpl) RenewalInfo(_ context.Context, response http.ResponseWriter, request *http.Request) {
```

Reply:
The context here is to satisfy the method signature of `wfeHandlerFunc`. The signature being:
```
type wfeHandlerFunc func(context.Context, http.ResponseWriter, *http.Request)
```